### DISCUSSION TITLE
保定-秋冬 | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by GZ-Metal-Cell** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#298] [Announcements]
# 保定-秋冬 | Zi-Zi's Journey

Zi-Zi's Journey

https://gz-metal-cell.github.io/galleries/%E7%A0%94%E7%A9%B6%E7%94%9F/%E6%B2%B3%E5%8C%97/%E4%BF%9D%E5%AE%9A-%E7%A7%8B%E5%86%AC/

**Comment by promefire** [comment]
我不是，我没有，不是我说的  :star2:

**Comment by GZ-Metal-Cell** [comment]
> @promefire
> 我不是，我没有，不是我说的  :star2:

工作日不在实验室搁这宿舍摸鱼呢

**Comment by promefire** [comment]
gitalk太垃圾了！！还得手动创建！！建议更换

**Comment by promefire** [comment]
博客里的相册[链接](https://gz-metal-cell.github.io/2023/12/30/Hexo-%E5%8D%9A%E5%AE%A2%E7%9B%B8%E5%86%8C%E5%8A%9F%E8%83%BD%E7%9A%84%E5%BC%80%E5%8F%91/)把后面句号也算上了！！！

**Comment by GZ-Metal-Cell** [comment]
> @promefire
> 博客里的相册[链接](https://gz-metal-cell.github.io/2023/12/30/Hexo-%E5%8D%9A%E5%AE%A2%E7%9B%B8%E5%86%8C%E5%8A%9F%E8%83%BD%E7%9A%84%E5%BC%80%E5%8F%91/)把后面句号也算上了！！！

可恶啊这个主题只支持 gitalk，还得再研究 😡

**Comment by GZ-Metal-Cell** [comment]
> @promefire
> 博客里的相册[链接](https://gz-metal-cell.github.io/2023/12/30/Hexo-%E5%8D%9A%E5%AE%A2%E7%9B%B8%E5%86%8C%E5%8A%9F%E8%83%BD%E7%9A%84%E5%BC%80%E5%8F%91/)把后面句号也算上了！！！

细啊 promefire
